crates/usvg-parser/src/filter.rs

```diff
@@ -44,37 +44,36 @@ pub(crate) fn convert(
     let mut has_invalid_urls = false;
     let mut filters = Vec::new();
 
-    let create_base_filter_func =
-        |kind, filters: &mut Vec<Rc<Filter>>, cache: &mut converter::Cache| {
-            // Filter functions, unlike `filter` elements, do not have a filter region.
-            // We're currently do not support an unlimited region, so we simply use a fairly large one.
-            // This if far from ideal, but good for now.
-            // TODO: Should be fixed eventually.
-            let rect = match kind {
-                Kind::DropShadow(_) | Kind::GaussianBlur(_) => {
-                    NonZeroRect::from_xywh(-0.5, -0.5, 2.0, 2.0).unwrap()
-                }
-                _ => NonZeroRect::from_xywh(-0.1, -0.1, 1.2, 1.2).unwrap(),
-            };
-
-            filters.push(Rc::new(Filter {
-                id: cache.gen_filter_id(),
-                units: Units::ObjectBoundingBox,
-                primitive_units: Units::UserSpaceOnUse,
-                rect,
-                primitives: vec![Primitive {
-                    x: None,
-                    y: None,
-                    width: None,
-                    height: None,
-                    // Unlike `filter` elements, filter functions use sRGB colors by default.
-                    color_interpolation: ColorInterpolation::SRGB,
-                    result: "result".to_string(),
-                    kind,
-                }],
-            }));
+    let create_base_filter_func = |kind, filters: &mut Vec<Rc<Filter>>| {
+        // Filter functions, unlike `filter` elements, do not have a filter region.
+        // We're currently do not support an unlimited region, so we simply use a fairly large one.
+        // This if far from ideal, but good for now.
+        // TODO: Should be fixed eventually.
```

Comment:
This code and some of the others looks like it is probably lifted from the upstream repo. Is that true? If so can you mark which code is lifted via a comment in GH?

Reply:
Yes, all of it is lifted from resvg's update from 0.36-0.37.

Reply:
All of it? We wrote nothing (baring the workflow changes?

Reply:
Yes, all of it. All I did was cherry-pick the changes so as not to pick changes that were incompatible (because the usvg tree structure changed between 0.36 and 0.37, so we couldn't pick those changes). I also made those changes in render.rs so that the tests would pass. I suspect that those changes were similarly made between 0.36 and 0.37, but I just manually did them because there were likely other changes that would have affected the tests in other ways.